AngularApp/src/app/diagnostic-data/components/detector-list/detector-list.component.ts

```diff
@@ -47,12 +51,26 @@ export class DetectorListComponent extends DataRenderBaseComponent {
 
     this._diagnosticService.getDetectors().subscribe(detectors => {
       this.detectorMetaData = detectors.filter(detector => this.renderingProperties.detectorIds.indexOf(detector.id) >=0);
-
       this.detectorViewModels = this.detectorMetaData.map(detector => this.getDetectorViewModel(detector));
 
+      let indexCount = 0;
       this.detectorViewModels.forEach((metaData, index) => {
         metaData.request.subscribe((response: DetectorResponse) => {
           this.detectorViewModels[index] = this.updateDetectorViewModelSuccess(metaData, response);
+
+          // Log all the children detectors
+          var childDetector = {
```

Comment:
The issue I had was with this part, just sort of a hacky way to log this information. Sent @cindylovescoding info about [forkjoin](https://www.learnrxjs.io/operators/combination/forkjoin.html) which I think is a cleaner solution.

@cindylovescoding  just make sure you see what happens when one of the detectors fails